src/lib.rs

```diff
@@ -16,6 +16,7 @@
 // It allows the Wasm Findex implementation to reuse the common traits
 // for searching and upserting indexes.
 #![feature(async_fn_in_trait)]
+#![feature(iter_next_chunk)]
```

Comment:
I know this is useful, but could we avoid this? It is not good for crypto crates to use unstable features.

Reply:
Since we already use nightly for `async_fn_in_trait` I prefer to have the most clean way to achieve de-serialization. (if we were on stable I would have use stable only). It's not really crypto stuff…

Reply:
Well it is inside a cryptographic repo. It's okay because I will move it to CloudproofRust later. But I still don't really like it :)